camel/agents/chat_agent.py

```diff
@@ -181,8 +196,8 @@ def update_messages(self, role: str,
         self.stored_messages.append(ChatRecord(role, message))
         return self.stored_messages
 
-    def submit_critic_choice(self, message: BaseMessage) -> None:
-        self.stored_messages.append(ChatRecord('assistant', message))
+    def submit_message(self, message: BaseMessage) -> None:
+        self.stored_messages.append(ChatRecord('user', message))
```

Comment:
it was 'assistant', now it's user. why is it so? was this a bug?

Reply:
This was a bug. All agent-external messages are considered LLM-user messages.

Reply:
Wait. Is it `'assistant'`? The message to be submitted is the response from OpenAI. So it would be `'assistant'` right?

Reply:
Right. It must be assistant. My bad.

Reply:
Reverted